qchem/setup.py

```diff
@@ -16,7 +16,14 @@
 with open("pennylane_qchem/_version.py") as f:
     version = f.readlines()[-1].split()[-1].strip("\"'")
 
-requirements = ["pennylane>=0.11", "openfermion", "openfermionpyscf", "openfermionpsi4", "pyscf<=1.7.1"]
+requirements = [
+    "pennylane>=0.11",
+    "scipy<1.5",
```

Comment:
How come `scipy` was pinned here?

Reply:
The latest version of PySCF doesn't support scipy>1.5. :(

This could cause potential dependency issues if anything in the stack requires scipy 1.5+, but so far all the tests seem to pass